mkdocs.yml

```diff
@@ -161,7 +161,7 @@ nav:
               - '1.x': products/access/release-notes/1.x.md
       - Self-styling:
           - 'Introduction': self-styling/index.md
-          - 'Onegini Template Styling': self-styling/Tool_Template_Styling.md
+#          - 'Onegini Template Styling': self-styling/Tool_Template_Styling.md
```

Comment:
Why did you put this in comments?

Reply:
yes, it is under discussion if the 'template engine' is part of self-styling, so for now I want to hide it... But completely removing felt like too rigorous...

Reply:
It's never gone, you can always revert a delete in git.